### DISCUSSION TITLE
Member Activity: Last Logged In / Last Active

### DISCUSSION THREAD

**Discussion by fv-tom** [community/community#134801] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Is there anyway to do an export of our Org and to see when a member was last active / last logged in. 

We need a way to easily see who is **not** needing a licence anymore and highlight anyway who really no longer needs access if they're not being active.

**Comment by davevad93** [comment]
Hi @fv-tom ,

I don't think GitHub natively supports exporting that kind of info, however, there's a `ruby` sample code from GitHub [platform-samples](https://github.com/github/platform-samples) repo to find inactive org users and export the list to a `csv` file. 

-  [Find Inactive Organization Members](https://github.com/github/platform-samples/tree/master/api/ruby/find-inactive-members)

**Reply by ayushkumar3344** [reply]
rat

**Comment by samus-aran** [comment, marked as answer]
We provide a [dormant user reports for Enterprise administrators](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/managing-dormant-users) but [note that there has been a retirement of `last_active` due to inaccurate data](https://github.blog/changelog/2025-09-14-last_active-field-removed-from-the-organization-members-export-report/).

You can also [utilise the Enterprise audit log and search for the specific user.](https://docs.github.com/en/enterprise-cloud@latest/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/searching-the-audit-log-for-your-enterprise)

**Reply by bwlang** [reply]
i think it's not great that github is no longer providing this information for standard orgs.
